changelogs/fragments/5249-add-new-channel-prefix.yml

```
@@ -0,0 +1,3 @@
+bugfixes:
+  - slack - fix message update for channels which starts with ``CP``. When ``message-id`` was passed it failed for channels wich starts with ``CP`` because of ``#`` simbol was added befor the ``channel_id``.(https://github.com/ansible-collections/community.general/pull/5249#issuecomment-1238520553).
+  - slack - docs improved. It wasn't clear that when `message_id` is supplied `channel_id` couldn't be a channel name. It has to be literal Channel ID returned in payload from API in previous task.
```

Docs changes do not need a changelog fragment.
```suggestion
  - slack - fix message update for channels which start with ``CP``. When ``message-id`` was passed it failed for channels which started with ``CP`` because the ``#`` symbol was added before the ``channel_id`` (https://github.com/ansible-collections/community.general/pull/5249).
```